tests/queries/tests.py

```diff
@@ -3798,3 +3798,11 @@ def test_ticket_23622(self):
             set(Ticket23605A.objects.filter(qy).values_list('pk', flat=True))
         )
         self.assertSequenceEqual(Ticket23605A.objects.filter(qx), [a2])
+
+
+class Ticket27397Tests(TestCase):
+    def test_ticket27397(self):
```

Comment:
@timgraham rather than use `class Ticket27397Tests(TestCase): ` i going to use https://github.com/django/django/blob/master/tests/queries/tests.py#L36 `class Queries1Tests(TestCase):` create with model `Order`,